test/strings/types.jl

```diff
@@ -72,20 +98,23 @@ b = IOBuffer()
 write(b, u)
 @test String(take!(b)) == ""
 
-str = "føøbar"
-u = SubString(str, 10, 10)
```

Comment:
I'd rather keep this test, just changing the indices. It's not clear what this tested, but it must have triggered a bug in a corner case.

Reply:
This comment still seems to apply.

Reply:
The test is retained but moved to line 95.